apps/test/unit/templates/MultipleSectionAssignerTest.js

```diff
@@ -151,29 +150,61 @@ describe('MultipleSectionsAssigner', () => {
       courseVersionId: assignedCourseANDUnitSection.courseVersionId,
     });
 
-    // Check that courses 1-6 have TeacherOptions and course 7 does not.
-    for (let i = 0; i < wrapper.instance().props.sections.length; i++) {
-      wrapper.instance().props.sections[i].participantType ===
```

Comment:
Unfortunately, the `instance()` method returns `null` for function components. https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/instance.html